src/pages/content/index.vue

```diff
@@ -1,72 +1,109 @@
 <template>
-	<section class="section">
-		<searchbar
-			v-model.lazy="searchQuery"
-			class="searchbar"
-			placeholder="Suche nach..."
-		/>
-		<pagination v-model="skippedItems" :state="pagination" />
-		<div class="columns">
-			<div v-for="content of searchResults" :key="content._id" class="column">
-				<pre>
-					{{ content }}
-				</pre
-				>
+	<section>
+		<div v-if="scrollY > 115" class="content__back-to-top">
+			<p @click="$_backToTop">
+				<i class="icon fa fa-chevron-circle-up"></i>
+			</p>
+		</div>
+		<div class="content">
+			<searchbar
+				v-model.lazy="searchQuery"
+				class="content__searchbar"
+				placeholder="Suche nach..."
+				:loading="loading"
+			/>
+			<p class="content__total">
+				<span v-if="searchQuery.length > 0">
+					{{ resources.total }} Suchergebnisse für "{{ searchQuery }}"
+				</span>
+				<span v-else> {{ resources.total }} resources </span>
+			</p>
+			<div v-if="resources.data.length === 0" class="content__no-results">
+				<content-empty-state />
+			</div>
+			<div class="content__cards-container">
+				<base-grid column-width="17rem">
+					<content-card
+						v-for="resource of resources.data"
+						:id="resource._id"
+						:key="resource._id"
+						class="card"
+						:content-category="resource.resourceCategory"
+						:description="resource.description"
+						:licenses="resource.licenses"
+						:mime-type="resource.mimeType"
+						:origin-id="resource.originId"
+						:provider-name="resource.providerName"
+						:tags="resource.tags.slice(0, 5)"
+						:thumbnail="resource.thumbnail"
+						:title="resource.title"
+						:url="resource.url"
+					/>
+				</base-grid>
 			</div>
+			<base-spinner
+				v-if="loading && resources.data.length !== 0"
+				class="content__spinner"
+				color="var(--color-primary)"
+			/>
 		</div>
-		<pagination v-model="skippedItems" :state="pagination" />
 	</section>
 </template>
 
 <script>
-import { mapGetters, mapState } from "vuex";
+import { mapState } from "vuex";
 import Searchbar from "@components/molecules/Searchbar";
-import Pagination from "@components/organisms/Pagination";
+import ContentCard from "@components/molecules/ContentCard";
+import ContentEmptyState from "@components/molecules/ContentEmptyState";
+import infiniteScrolling from "@mixins/infiniteScrolling";
+import BaseGrid from "@components/base/BaseGrid";
 
 export default {
 	components: {
 		Searchbar,
-		Pagination,
+		ContentCard,
+		ContentEmptyState,
+		BaseGrid,
+	},
+	mixins: [infiniteScrolling],
```

Comment:
- [ ] please use the full width layout
```suggestion
	mixins: [infiniteScrolling],
	layout: "loggedInFull",
```